lib/web_ui/test/engine/image/image_test.dart

```diff
@@ -19,6 +19,52 @@ Future<void> testMain() async {
   // TODO(polina-c): unskip the test when bug is fixed:
   // https://github.com/flutter/engine/pull/35791
   }, skip: true);
+
+  test('Image constructor invokes onCreate once', () async {
+    int onCreateInvokedCount = 0;
+    ui.Image? createdImage;
+    ui.Image.onCreate = (ui.Image image) {
+      onCreateInvokedCount++;
+      createdImage = image;
+    };
+
+    final ui.Image image1 = await _createImage();
+
+    expect(onCreateInvokedCount, 1);
+    expect(createdImage, image1);
+
+    final ui.Image image2 = await _createImage();
+
+    expect(onCreateInvokedCount, 2);
+    expect(createdImage, image2);
+
+    ui.Image.onCreate = null;
+  // TODO(polina-c): unskip the test when bug is fixed:
+  // https://github.com/flutter/engine/pull/35791
```

Comment:
https://github.com/flutter/engine/pull/35791 is now merged

Reply:
It was wrong URL. Replaced with right one. The issue is not fixed yet.